extensions/amp-image-lightbox/0.1/amp-image-lightbox.js

```diff
@@ -744,21 +745,15 @@ class AmpImageLightbox extends AMP.BaseElement {
     this.container_.appendChild(this.captionElement_);
 
     // Invisible close button at the end of lightbox for screen-readers.
-    const screenReaderCloseButton = this.element.ownerDocument
-        .createElement('button');
     // TODO(aghassemi, #4146) i18n
-    screenReaderCloseButton.textContent = 'Close the lightbox';
-    screenReaderCloseButton.classList.add('i-amphtml-screen-reader');
-    // This is for screen-readers only, should not get a tab stop.
-    screenReaderCloseButton.tabIndex = -1;
-    screenReaderCloseButton.addEventListener('click', () => {
+    addScreenReaderButton(this.element, 'Close the lightbox', () => {
       this.close();
     });
-    this.element.appendChild(screenReaderCloseButton);
 
     const gestures = Gestures.get(this.element);
     this.element.addEventListener('click', e => {
       if (!this.entering_ &&
+            !e.target.classList.contains('i-amphtml-screen-reader') &&
```

Comment:
It wasn't causing issues, it just caused an unexpected result in a test. But I felt a double trigger might have unexpected consequences later. I could add an `isScreenReaderButton` to hide that implementation detail, or if the double trigger isn't a problem I can just remove this.